debian/entry.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
+function remove_buildtime_env_var()
+{
+	unset QEMU_CPU
```

Comment:
@petrosagg, I think on device QEMU will no longer emulate and this command will report correct information without this env var